yambo-front/src/components/CardCreate.tsx

```diff
@@ -220,34 +228,67 @@ const CardCreate: FC = () => {
     });
   };
 
+  const handleSave = (e: MouseEvent) => {
```

Comment:
Is there a reason why you're using an <a> tag for a button input? The "Link" tag is a styled <a> tag. Also, I'd prefer calling the styled button component something else as react-router-dom uses the <Link> tag.

Reply:
I guess Github closes native HTML in comments. I mean the `<a>` element is used as a regular button, and react-router-dom uses the `<Link>` element so I'd prefer calling it something different.

Reply:
Since these are just pieces of text that when clicked will change state on the page, maybe I should just make them a styled.div.

I will change the styled component name from "Link" and remove the <a> altogether.